lib/fluent/plugin/in_monitor_agent.rb

```diff
@@ -330,7 +282,7 @@ def all_plugins
     # from the plugin `pe` recursively
     def self.collect_children(pe, array=[])
       array << pe
-      if pe.is_a?(MultiOutput) && pe.respond_to?(:outputs)
```

Comment:
It is required to collect data for old MultiOutput plugins.
Old (v0.12) MultiOutput plugin are not children of `Fluent::Plugin::MultiOutput`, but `Fluent::Compat::MultiOutput` (and it is child of `Fluent::Plugin::BareOutput`).
fluent-plugin-forest and other plugins are affected with it.